webapp/app/partials/perf/perfctrl.html

```diff
@@ -37,6 +37,10 @@
     </select>
     <hr/>
     <div id="graph"></div>
+    <div id="graph-bottom" ng-show="showGraphBottom">
```

Comment:
To remove the need for the showGraphBottom function, just put the expression seriesList.length > 0 in ng-show instead.